server/src/main/scala/scaladex/server/service/SchedulerService.scala

```diff
@@ -1,57 +1,55 @@
 package scaladex.server.service
 
-import java.time.Instant
-
-import scala.concurrent.Await
 import scala.concurrent.ExecutionContext
 import scala.concurrent.ExecutionContextExecutor
 import scala.concurrent.Future
-import scala.concurrent.duration.Duration
-import scala.util.control.NonFatal
+import scala.concurrent.duration._
 
-import ch.epfl.scala.services.DatabaseApi
+import ch.epfl.scala.index.newModel.NewProject
+import ch.epfl.scala.services.SchedulerDatabase
 import ch.epfl.scala.utils.ScalaExtensions._
 import com.typesafe.scalalogging.LazyLogging
+import scaladex.server.service.SchedulerService._
+import scaladex.template.SchedulerStatus
+
+class SchedulerService(db: SchedulerDatabase) extends LazyLogging {
+  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
+
+  private val mostDependentProjectScheduler = new Scheduler("project-trends", mostDependentProjectJob, 1.hour)
+  private val mostRecentProject = new Scheduler("update-projects", updateProjectJob, 30.minutes)
```

Comment:
It would be better to have the same name for the variable and the scheduler.

```suggestion
  private val mostDependentProjectScheduler = new Scheduler("most-dependent", mostDependentProjectJob, 1.hour)
  private val updateProject = new Scheduler("update-projects", updateProjectJob, 30.minutes)
```